File: Python/URI/1037.py
a = float(input());

if a >= 0.00 and a <= 25.00:
    print("Intervalo [0,25]")
elif a > 25.00 and a <= 50.00:
   print("Intervalo (25,50]")
elif a > 50.00 and a <= 75.00:
    print("Intervalo (50,75]")
elif a > 75.00 and a <= 100.00:
    print("Intervalo (75,100]")
else:
    print("Fora de intervalo")

File: JS/URI/1037.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split(' ');

var a = parseFloat(lines.shift());

if (a >= 0.00 && a <= 25.00)
    console.log("Intervalo [0,25]");
else if (a > 25.00 && a <= 50.00)
    console.log("Intervalo (25,50]");
else if (a > 50.00 && a <= 75.00)
    console.log("Intervalo (50,75]");
else if (a > 75.00 && a <= 100.00)
    console.log("Intervalo (75,100]");
else
    console.log("Fora de intervalo");